lms/static/scripts/frontend_apps/utils/test/api-test.js

```diff
@@ -272,4 +291,53 @@ describe('api', () => {
     assert.deepEqual(firstCallResult, await fakeResponse.json());
     assert.deepEqual(secondCallResult, await fakeResponse.json());
   });
+
+  it('aborts if fetch is canceled', async () => {
```

Comment:
Though I understand this test's contents, I'm having a little trouble mapping them to this description. 

As I understand from docs, when `AbortController.abort` is called, a `fetch` operation with a corresponding `signal` to that controller will reject with a `DOMException` that looks like the one generated here. As such, this is what an aborted `fetch` would "look like" to a caller. 

OTOH, I'm not clear where the "canceled" piece comes in here. That is, where is the line between "cancel" and "abort"?

Reply:
> As such, this is what an aborted fetch would "look like" to a caller.

Yes, indeed.

> OTOH, I'm not clear where the "canceled" piece comes in here. That is, where is the line between "cancel" and "abort"?

"Canceled" and "aborted" are synonyms in this context. I'll revise the wording in this description.